app/components/candidate_interface/referees_review_component.rb

```diff
@@ -26,7 +26,13 @@ def minimum_references
     end
 
     def show_missing_banner?
-      !@application_form.references_completed && @editable if @submitting_application
+      !@application_form.references_completed && @editable && @submitting_application
+    end
+
+    def show_references_not_provided?
+      return false unless FeatureFlag.active?(:decoupled_references)
+
+      !@application_form.enough_references_have_been_provided? && @editable && @submitting_application
```

Comment:
FYI, I'm writing a new component to replace the existing referees review one. I've made a note on my card to move this over when it's ready.